samples/toolbarconfigurator/js/toolbartextmodifier.js

```diff
@@ -265,7 +265,7 @@
 
 		this.codeContainer.refresh();
 
-		this.hintContainer.setHtml( header + '<dl>' + listHeader + unusedElements + '</dl>' );
+		this.hintContainer.setHtml( CKEDITOR.tools.htmlSafeByReview( header + '<dl>' + listHeader + unusedElements + '</dl>', 'templated constants') );
```

Comment:
`header` and `listHeader` are constant values, agreed, but `unusedElements` are not, essentially coming from https://github.com/youssef-attia/ckeditor4/blob/45995cb4173614e5582c2954573a2c147d571e74/samples/toolbarconfigurator/js/toolbartextmodifier.js#L302 before being string-processed.

On one hand, it probably looks like we could trust it because it looks like it was already on the page, but on the other hand, maybe it is safer to legacy convert this bit?

Reply:
samples/ removed

Reply:
LGTM